getting-started.md

```diff
@@ -26,9 +27,12 @@ Getting started with TimescaleDB involves a few steps:
 1. [Creating a standard PostgreSQL database and enabling the TimescaleDB extension on the database.][setup]
 1. [Creating a hypertable to store your data.][create-hypertables]
 1. [Migrating your data to a hypertable][migrate] (optional)
+1. [Setting up data node authentication for distributed hypertables][data-node-authentication]
 
 [hypertables]: /introduction/architecture#hypertables
 [install]: /getting-started/installation
 [setup]: /getting-started/setup
 [create-hypertables]: /getting-started/creating-hypertables
+[create-distributed-hypertables]: /getting-started/creating-distributed-hypertables
```

Comment:
Is there a reference to this link somewhere in the main text?

Reply:
Nope, temporarily added another list item but forgot to remove this one.